advisories/published/2025/JLSEC-0000-mnsaputow-8du40t.md

```diff
@@ -0,0 +1,24 @@
+```toml
+schema_version = "1.7.3"
+id = "JLSEC-0000-mnsaputow-8du40t"
+modified = 2025-10-22T22:35:33.968Z
+upstream = ["CVE-2021-36647"]
+references = ["https://github.com/ARMmbed/mbedtls/releases/", "https://kouzili.com/Load-Step.pdf", "https://tls.mbed.org/tech-updates/security-advisories/mbedtls-security-advisory-2021-07-1", "https://github.com/ARMmbed/mbedtls/releases/", "https://kouzili.com/Load-Step.pdf", "https://tls.mbed.org/tech-updates/security-advisories/mbedtls-security-advisory-2021-07-1"]
+
+[[affected]]
+pkg = "MbedTLS_jll"
+ranges = ["< 2.27.0+0", ">= 2.28.0+0"]
```

Comment:
NVD has a misreading of [the original report](https://web.archive.org/web/20211228163530/https://tls.mbed.org/tech-updates/security-advisories/mbedtls-security-advisory-2021-07-1).

> Affected users will want to upgrade to Mbed TLS 3.0.0, 2.27.0 or 2.16.11 depending on the branch they're currently using.

This got whispered down the grapevine to "all versions before 3.0.0, 2.27.0 or 2.16.11" in NVD's description, which led to: 

<img width="708" height="225" alt="image" src="https://github.com/user-attachments/assets/4e0d7a9e-98b4-4292-bf8c-4d973f8ea6b0" />

```suggestion
ranges = ["< 2.27.0+0"]
```